src/Numerics/Optimizers/ExhaustiveOptimizer/exhaustiveoptimizer.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+#=========================================================================
+#
+#  Copyright NumFOCUS
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#
+
+# Python example demonstrating itk.ExhaustiveOptimizer usage
+import sys
+from math import pi
+
+import itk
+
+EXIT_SUCCESS = 0
+EXIT_FAILURE = 1
+
+def main(argv):
+    if(len(argv) < 3):
+        raise Exception(f'Usage: {argv[0]} fixedImage movingImage')
+
+    dimension = 2
+    FixedImageType = itk.Image[itk.F, dimension]
+    MovingImageType = itk.Image[itk.F, dimension]
+    TransformType = itk.Euler2DTransform[itk.D]
+    OptimizerType = itk.ExhaustiveOptimizerv4[itk.D]
+    MetricType = itk.MeanSquaresImageToImageMetricv4[FixedImageType, MovingImageType]
+    TransformInitializerType = \
+        itk.CenteredTransformInitializer[itk.MatrixOffsetTransformBase[itk.D,2,2],
+                                         FixedImageType, MovingImageType]
+    RegistrationType = itk.ImageRegistrationMethodv4[FixedImageType,MovingImageType]
+
+    fixedImage = itk.imread(argv[1], itk.F)
+    movingImage = itk.imread(argv[2],itk.F)
+    transform = TransformType.New()
```

Comment:
We can use the single line:

```
transform = itk.Euler2DTransform[itk.D].New()
```

Reply:
Common convention for cxx and Python examples alike seems to be defining the type and instantiating on separate lines for readability. Is it ok to break that convention here?

Reply:
Currently the ITKExamples are inconsistent -- we had tried to use the same naming convention at some time in the past, and some are that way. However, we should make the repository consistently use idiomatic Python, and some have been updated in this way. I created this issue to track this: https://github.com/InsightSoftwareConsortium/ITKExamples/issues/199